### PR TITLE
Upgrade IRISHub to v2.1

### PR DESCRIPTION
**Pull request by taramakage**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,26 @@
...
 ## [Unreleased]
 
+### API Breaking
+
+* (Auth) Query for BaseAccount type account is no longer supported and is replaced by EthAccount type. 
+
+### State Machine Breaking
+
+* (IRISHub) [\#2863](https://github.com/irisnet/irishub/pull/2863) Bump up tibc-go version to xxx
+* (IRISHub) [\#2863](https://github.com/irisnet/irishub/pull/2863) Bump up irismod version to xxx
```
Comment (dreamer-zq) [inline CHANGELOG.md:12]: what is "xxx" ?

File: app/modules.go [hunk 2]
```diff
@@ -74,6 +75,8 @@ import (
 	tibchost "github.com/bianjieai/tibc-go/modules/tibc/core/24-host"
 	tibccli "github.com/bianjieai/tibc-go/modules/tibc/core/client/cli"
 
+	nfttransfer "github.com/bianjieai/nft-transfer"
+
```
Comment (dreamer-zq) [inline app/modules.go:79]: format code

File: CHANGELOG.md (hunk 1 above)
Comment (taramakage) [reply CHANGELOG.md:12]: Tags of these dependencies are not decided yet, so they are temporarily replaced with "xxx".

File: app/app.go [hunk 3]
```diff
@@ -212,6 +228,15 @@ func NewIrisApp(
...
 
+	// Setup Mempool
+	baseAppOptions = append(baseAppOptions, func(app *baseapp.BaseApp) {
+		memPool := mempool.NoOpMempool{}
+		app.SetMempool(memPool)
+		handler := baseapp.NewDefaultProposalHandler(memPool, app)
+		app.SetPrepareProposal(handler.PrepareProposalHandler())
+		app.SetProcessProposal(handler.ProcessProposalHandler())
+	})
+
```
Comment (dreamer-zq) [inline app/app.go:239]: `
func (app *baseapp.BaseApp) {
	memPool := mempool.NoOpMempool{}
	app.SetMempool(memPool)
	handler := baseapp.NewDefaultProposalHandler(memPool, app)
	app.SetPrepareProposal(handler.PrepareProposalHandler())
	app.SetProcessProposal(handler.ProcessProposalHandler())
}
`
Refined into methods to increase readability

**Review by dreamer-zq — APPROVED** [review]

**Review by dreamer-zq — APPROVED** [review]

**Pull request merged**